build/helper/codegen_helper.py

```diff
@@ -123,9 +145,19 @@ class ParamListType(Enum):
     'skip_session_handle': False,
     'skip_output_parameters': False,
     'skip_ivi_dance_size_parameter': False,
+    'reordered_for_default_values': False,
     'session_name': 'vi',
     'name_to_use': 'ctypes_type_library_call',
 }
+ParamListTypeDefaults[ParamListType.LIBRARY_IMPL_METHOD] = {
+    'skip_self': False,
+    'skip_session_handle': False,
+    'skip_output_parameters': False,
+    'skip_ivi_dance_size_parameter': False,
+    'reordered_for_default_values': False,
+    'session_name': 'vi',
+    'name_to_use': 'python_name',
+}
 
 
 def get_params_snippet(function, param_type, options={}):
```

Comment:
This function is 50 lines long... we should consider breaking up at some point (not needed yet, but an observation)